File: archives/C#/0771. Jewels and Stones.cs
public class Solution {
    public int NumJewelsInStones(string J, string S) {
        HashSet<char> JSet=new HashSet<char>(J.ToCharArray());
        int cnt=0;
        foreach(char s in S){
            if(JSet.Contains(s)){
                cnt+=1;
            }
        }
        return cnt;
    }
}

File: archives/20190519python/0771. Jewels and Stones.py
class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        J_set=set()
        for j in J:
            J_set.add(j)
        cnt=0
        for s in S:
            if s in J_set:
                cnt+=1
        return cnt
